docs/docs/integrations/chat/llamacpp.ipynb

```diff
@@ -0,0 +1,448 @@
+{
+ "cells": [
```

Comment:
Add ChatLlamaCpp to `__init__.py` so that we can import per usual:

```
from langchain_community.chat_models import ChatLlamaCpp
```

Reply:
Bump on this. Make changes to `__init__.py` as shown here:
https://github.com/langchain-ai/langchain/commit/4cfadef23575952f0988570e8ec47d1d602b4f20#diff-c3d21dcf5ebce452afbee03a028b6ba779bf6e7a871545ce25574cf70b79ecc0